src/cadenza.js

```diff
@@ -432,6 +432,20 @@ export class CadenzaClient {
     return this.#show(resolvePath(backgroundMapView), params, signal);
   }
 
+  /**
+   * Set the visibility of a layer in an embedded map view. The map view must be shown prior to calling this.
+   * For layers in groups, the layer path contains the ancestors also. When making a layer visible, its ancestors will
+   * be made visible also. When hiding a layer, the ancestors are not affected. The operation is asynchronous and its
+   * completion is signaled by an event of type 'setLayerVisibility:success' or 'setLayerVisibility:error'.
+   *
+   * @param {WorkbookLayerPath} layer - The layer to show or hide, identified by its path
+   * @param {boolean} visible - The visibility state of the layer
+   */
+  setLayerVisibility(layer, visible = true) {
```

Comment:
I find a setter with a default for the value surprising. If you want to keep it, then you need to make the param optional in the JSDoc:
```
@param {WorkbookLayerPath} [layer=true] layer - ...
```
(Here the JSDoc is used for the TS types.)

Reply:
Actually, I changed my mind about it , will make it non-optional.